packages/gatsby-remark-images/src/__tests__/gastby-browser.js

```diff
@@ -0,0 +1,37 @@
+const {
+  imageClass,
+  imageWrapperClass,
+  imageBackgroundClass,
+} = require(`../constants`)
+const { onRouteUpdate } = require(`../gatsby-browser`)
+
+const createImageElement = () => {
+  global.document.body.innerHTML = `
+    <span class="${imageWrapperClass}">
+      <span class="${imageBackgroundClass}"></span>
+      <img class="${imageClass}">
+    </span>
+  `
+
+  return global.document.querySelector(`.${imageClass}`)
+}
+
+test(`it sets correct box-shadow on img element`, () => {
+  const pluginOptions = {}
+  const imageElement = createImageElement()
+
+  onRouteUpdate({}, pluginOptions)
+  imageElement.dispatchEvent(new Event(`load`))
+
+  expect(imageElement.style[`box-shadow`]).toBe(`inset 0px 0px 0px 400px white`)
+})
```

Comment:
@birjolaxew [mentioned that it'd be nice to not rely on a 400px `box-shadow`](https://github.com/gatsbyjs/gatsby/issues/15676#issuecomment-511743350), since it won't work for transparent images more than 800px wide. Any thoughts on handling that?

- Could set the `box-shadow` to `options.maxWidth / 2`, but I'm not sure if that'd work for images that are taller than they are wide.
- Could use `background-color` instead of `box-shadow`, which seems to work for me, but there may be implications I'm unaware of.

Reply:
Let's continue this discussion in a new issue!